locale-source/locale/tr/LC_MESSAGES/djangojs.po

```diff
@@ -1126,8 +1134,8 @@ msgstr ""
 
 #: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:221
 #, javascript-format
-msgid "'s {}comment{} has been reported 1 time since it's creation"
-msgid_plural "'s {}comment{} has been reported %s times since it's creation"
+msgid "This {}comment{} has been reported 1 time since it's creation"
```

Comment:
I think correct grammar is `its creation` or `it's created`. Can you check with @philli-m ?